pkg/config/config.go

```diff
@@ -147,6 +147,7 @@ type Install struct {
 	TTY       string `json:"tty,omitempty"`
 	ForceGPT  bool   `json:"forceGpt,omitempty"`
 	Role      string `json:"role,omitempty"`
+	EnableSFTP bool   `json:"enableSftp,omitempty"`
```

Comment:
I suggest we design the flag like this:
```yaml
os:
  sshd:
     sftp: true
```

This provides the opportunity to add more sshd configuration in the future. The reason to put it under `os` is the configuration is supposed to take effect after the installation (in real OS).

Reply:
Sure, I move it into the sshd components.
Updated. Please help to recheck!